APIManager.js

```diff
@@ -0,0 +1,81 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+
+    constructor() {
+        this.data = {
+            user: {},
+            quote: {},
+            pokemon: {},
+            aboutme: {},
+            friends: {}
+        }
+
+    }
+    randomFriends = function () {
+        $.ajax({
+            url: 'https://randomuser.me/api/?results=5',
+            dataType: 'json',
+            success: (response) => {
+                this.data.friends.friend1first = response.results[0].name.first
```

Comment:
it would be better to use the `.map()` method so the friends are not hardcoded. By using the map method you would be able to increase the friends array if you wanted without it causing problems.